components/ui/icons/src/main/res/drawable/mozac_ic_back.xml

```diff
@@ -2,13 +2,6 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="24dp"
-        android:height="24dp"
-        android:viewportWidth="24.0"
-        android:viewportHeight="24.0"
-        android:autoMirrored="true">
-    <path
-        android:fillColor="#ffffff"
-        android:pathData="M21,11H5.914l5.793,-5.793a1,1 0,0 0,-1.414 -1.414l-7.5,7.5a1,1 0,0 0,0 1.414l7.5,7.5a1,1 0,0 0,1.414 -1.414L5.914,13H21a1,1 0,0 0,0 -2Z"/>
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="24dp" android:height="24dp" android:viewportWidth="24" android:viewportHeight="24">
+    <path android:fillColor="#F9F9FA" android:pathData="M11.707 3.293c0.39 0.39 0.39 1.024 0 1.414L5.414 11H21c0.552 0 1 0.448 1 1s-0.448 1-1 1H5.414l6.293 6.293c0.39 0.39 0.39 1.024 0 1.414-0.39 0.39-1.024 0.39-1.414 0l-8-8c-0.39-0.39-0.39-1.024 0-1.414l8-8c0.39-0.39 1.024-0.39 1.414 0z"/>
 </vector>
```

Comment:
Mh, the new versions have a different fill color. This may break some of the apps that are already using them (assuming they haven't tinted them).

I wonder whether we should create a `color` for this. Then apps can override the color without having to tint all the icons they load.